src/Controls/tests/UITests/Tests/Issues/Issue18716.cs

```diff
@@ -0,0 +1,28 @@
+﻿using NUnit.Framework;
+using UITest.Appium;
+using UITest.Core;
+
+namespace Microsoft.Maui.AppiumTests.Issues
+{
+	public class Issue18716 : _IssuesUITest
+	{
+		public Issue18716(TestDevice device)
+			: base(device)
+		{ }
+
+		public override string Issue => "Touch events are not working on WebView when a PDF is displayed";
+
+		[Test]
+		[Category(UITestCategories.WebView)]
+		public async Task CanScrollWebView()
+		{
+			this.IgnoreIfPlatforms(new TestDevice[] { TestDevice.Android, TestDevice.Mac, TestDevice.Windows });
```

Comment:
Would this be worth enabling for all platforms?

Reply:
Enabled on more platforms. Scrolling using gestures, so Android and iOS.